src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+interface Props {
+  selectedTodo: Todo;
+  onClose: (todo: Todo | null) => void;
+}
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, onClose }) => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [isModalLoading, setIsModalLoading] = useState(false);
```

Comment:
```suggestion
  const [isModalLoading, setIsModalLoading] = useState(true);
```
to fix blinking when you open the modal